src/status_im/contexts/wallet/send/utils.cljs

```diff
@@ -47,3 +65,104 @@
                (assoc acc k (if (money/equal-to v 0) "<0.01" v)))
              {}
              amounts))
+
+(defn token-valid-networks
```

Comment:
Is this a list of networks that has positive balance and not disabled?

Reply:
I renamed this function to be more understandable, now is `token-available-networks-for-suggested-routes`

Reply:
But basically adds all the chains that this token is available (should count those that balance is zero too), and filters the chains that are disabled.